web/cypress/integration/route/search-route.spec.js

```diff
@@ -17,7 +17,7 @@
 /* eslint-disable */
 
 context('Create and Search Route', () => {
-  const timeout = 500;
+  const timeout = 5000;
```

Comment:
why change this？

Reply:
>Similar to #2059

see this,we can't pass front-end test in CI without changing this